akvo/rsr/management/commands/recalculate_project_aggregation.py

```diff
@@ -19,11 +22,17 @@ def handle(self, *args, **options):
             return
 
         descendants = project.descendants()
-        periods = IndicatorPeriod.objects\
-            .annotate(approved_count=Count('data', filter=Q(data__status=IndicatorPeriodData.STATUS_APPROVED_CODE)))\
-            .filter(approved_count__gte=1, indicator__result__project__in=descendants)
-
+        periods = IndicatorPeriod.objects.filter(indicator__result__project__in=descendants)
+        if not options.get("all"):
+            periods = periods.annotate(
+                approved_count=Count('data', filter=Q(data__status=IndicatorPeriodData.STATUS_APPROVED_CODE))
+            ).filter(approved_count__gte=1)
+            if options.get("quantitative"):
+                periods = periods.filter(indicator__type=QUANTITATIVE)
```

Comment:
Can we even aggregate qualitative jobs? Maybe `schedule_aggregation_jobs` should include the check for a quantitative indicator? The a `select_related` for the indicator would be necessary.

Otherwise, I think that it should be possible to filter all periods by qualitative indicators. This would give the user the option to schedule jobs for all periods, or all quantitative periods , quantitative periods with approved updates, periods with approved updates.

Reply:
Will be done in a separate PR